CONTRIBUTING.md

```diff
@@ -31,9 +31,14 @@ https://github.com/pymovements/pymovements/issues/new/choose) or write us at
 
 ## Code of Conduct
 
-Everyone participating in the pymovements project, and in particular in our issue tracker and pull
-requests, is expected to treat other people with respect and more generally to follow the guidelines
-articulated in the [Python Community Code of Conduct](https://www.python.org/psf/codeofconduct/).
+Everyone participating in the pymovements project, including in issues, pull requests,
+discussions, and any other community spaces, is expected to treat others with respect
+and to contribute to an open, welcoming, and inclusive environment.
+
+Please read and follow our [Code of Conduct](CODE_OF_CONDUCT.md) for full details.
+
+Our community guidelines build on the principles of the
+[Python Community Code of Conduct](https://www.python.org/psf/codeofconduct/).
```

Comment:
you can remove this sentence as it's already included in the code of conduct document.